packages/terra-list/tests/wdio/multi-select-list-spec.js

```diff
@@ -0,0 +1,209 @@
+/* global before, browser, Terra */
+
+describe('Multi Select List', () => {
+  before(() => browser.setViewportSize(Terra.viewports('medium')[0]));
+
+  describe('Multi Select List Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list'));
+
+    Terra.should.matchScreenshot();
+
+    it('should highlight the first item upon clicking', () => {
+      browser.click('ul li:nth-child(1)');
+    });
+    Terra.should.matchScreenshot('first item highlighted');
+
+    it('should highlight the second item upon clicking', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('first and second item highlighted');
+
+    it('should highlight the third item upon clicking', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('first second and third item highlighted');
+
+    it('should unhighlight the second item upon clicking', () => {
+      browser.click('ul li:nth-child(2)');
+    });
+    Terra.should.matchScreenshot('first and third item highlighted');
+
+    it('should unhighlight the third item upon clicking', () => {
+      browser.click('ul li:nth-child(3)');
+    });
+    Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+
+    describe('Enter Key', () => {
+      before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list'));
+
+      it('should select and highlight the first item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first item highlighted');
+
+      it('should highlight the second item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first and second item highlighted');
+
+      it('should highlight the third item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first second and third item highlighted');
+
+      it('should unhighlight the second item upon enter', () => {
+        browser.keys(['Shift', 'Tab', 'Enter', 'Shift']);
+      });
+      Terra.should.matchScreenshot('first and third item highlighted');
+
+      it('should unhighlight the third item upon enter', () => {
+        browser.keys(['Tab', 'Enter']);
+      });
+      Terra.should.matchScreenshot('first item highlighted third item unhighlighted');
+    });
+
+    describe('Space Bar', () => {
+      before(() => browser.url('/#/raw/tests/terra-list/list/multi-select-list/default-multi-select-list'));
+
+      it('should highlight the selected items upon space keydown', () => {
```

Comment:
"highlight" seems like the wrong word here reading this interaction. Is this selecting the first item on space or just indication the item has focus without selection?